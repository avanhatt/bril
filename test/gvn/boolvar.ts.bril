test/ts/boolvar.ts.bril