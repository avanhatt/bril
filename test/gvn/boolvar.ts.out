void main() {
b1:
  v01: bool = const true;
  x1: bool = id v01;
  br x1 then.1 else.1;
then.1:
  v31: int = const 1;
  print v31;
  jmp endif.1;
else.1:
  v51: int = const 2;
  print v51;
  jmp endif.1;
endif.1:
  ret ;
}
